modules/internal/LocaleModelHelpRuntime.chpl

```diff
@@ -78,6 +78,7 @@ module LocaleModelHelpRuntime {
 
   pragma "insert line file info"
   pragma "always resolve function"
+  pragma "codegen for CPU and GPU"
```

Comment:
Two notes that are more of a braindump:
- I think it is reasonable to think whether "always resolve function" should imply "codegen for CPU and GPU" with the GPU locale model. Functions with the former pragma cannot be generic, so they can be resolved even without a call to them. Sort of like an `export` function. But for some functions like this one, we need the pragma, because the call to these functions are injected by the compiler at codegen time. Normally, if there's no call to a function at resolve time, we don't resolve that function and prune it from the tree.
- An alternative solution to this (given that it is calling an empty device function), would have been adding `no GPU codegen` on it. But as you already need get/put for some mysterious reason and can't tack that pragma on that, it makes sense to make all the changes in the same approach. Just wanted to mention that.